src/main/java/org/testng/SkipException.java

```diff
@@ -6,11 +6,10 @@
  * return of {@link #isSkip()}.
  * Users may provide extensions to this mechanism by extending this class.
  *
- * @author <a href='mailto:the[dot]mindstorm[at]gmail[dot]com'>Alex Popescu</a>
  * @since 5.6
  */
 public class SkipException extends RuntimeException {
-  private static final long serialVersionUID = 4052142657885527260L;
```

Comment:
Shouldn't we be leaving this as is, because `Exception`(s) are serializable and I thought we should be adding serial ids else sonar cube complains ?